src/main/scala/org/scanet/optimizers/Nadam.scala

```diff
@@ -0,0 +1,27 @@
+package org.scanet.optimizers
+
+import org.scanet.core._
+import org.scanet.syntax._
+
+case class Nadam(rate: Output[Float], beta1: Output[Float], beta2: Output[Float], epsilon: Output[Float]) extends Algorithm {
```

Comment:
Could you also rename:
- `board/AdagGrad` to `board/AdaGrad`
- `board/AdagDelta` to `board/AdaDelta`
I made a typo...